jib-cli/README.md

```diff
@@ -9,13 +9,29 @@ a Java library for building containers without Docker.
 This CLI tool is _experimental_ and its options and structure
 are almost certain to change.
 
+## Table of Contents
+* [Get the Jib CLI](#get-the-jib-cli)
+  * [Download a java application](#download-a-java-application)
+  * [Build yourself from source](#build-yourself-from-source)
+* [Supported Commands](#supported-commands)
+* [Build](#build)
+  * [Quickstart](#quickstart)
+  * [Fully Annotated `jib.yaml`](#fully-annotated-jibyaml)
+  * [Options](#options)
+* [Jar](#jar)
+  * [Options](#options)
+* [Common Jib CLI Options](#common-cli-options)
+  * [Auth/Security](#authsecurity)
+  * [Info Params](#info-params)
+  * [Debugging Params](#debugging-params)
+
 ## Get the Jib CLI
 
 ### Download a java application
 
 A JRE is required to run this Jib CLI distribution.
 
-Find the latest jib-core release on the [Releases page](https://github.com/GoogleContainerTools/jib/releases), download `jib-jre-<version>.zip`, and unzip it. The zip file contains the `jib` (`jib.bat` for Windows) script at `jib/bin/`. Optionally, add the binary directory to your `$PATH` so that you can call `jib` from anywhere.
+Find the latest jib-cli release on the [Releases page](https://github.com/GoogleContainerTools/jib/releases), download `jib-jre-<version>.zip`, and unzip it. The zip file contains the `jib` (`jib.bat` for Windows) script at `jib/bin/`. Optionally, add the binary directory to your `$PATH` so that you can call `jib` from anywhere.
```

Comment:
Nice catch. Perhaps from now on, for easy access, we should link the latest release and update it whenever we make a new release. For example,
```
Find the [latest jib-cli release 0.2.0](...) on the [Releases page](https://github.com/GoogleContainerTools/jib/releases), download `jib-jre-<version>.zip`, and unzip it. ...
```

Reply:
Good idea.